Update /web/css/system-color/index.md (#26599)
mdn/content: files/en-us/web/css/system-color/index.md

- `-moz-ButtonHoverFace` {{non-standard_inline}}
  - : The background color of a button that the mouse pointer is over (which would be `ThreeDFace` or `ButtonFace` when the mouse pointer is not over it). Should be used with the `-moz-ButtonHoverText` foreground color.
- `-moz-ButtonHoverText` {{non-standard_inline}}
  - : The text color of a button that the mouse pointer is over (which would be ButtonText when the mouse pointer is not over it). Should be used with the `-moz-ButtonHoverFace` background color.
- `-moz-CellHighlight` {{non-standard_inline}}
  - : Background color for selected item in a tree widget. Should be used with the `-moz-CellHighlightText` foreground color. See also `-moz-html-CellHighlight`.

> - `-moz-CellHighlightText` {{non-standard_inline}}

Reviewer: The line right below this one also needs fixing

  - : Background color for dialog boxes. Should be used with the `-moz-DialogText` foreground color.
- `-moz-DialogText` {{non-standard_inline}}
  - : Text color for dialog boxes. Should be used with the `-moz-Dialog` background color.
- `-moz-dragtargetzone` {{non-standard_inline}}

>   - : Specify the areas where an element can be dropped during drag and drop operations.

Reviewer 2: > Could you add a link (here in the review) to support this description? That would be quite helpful.

Of course. I will do it.

>   - : Specify the areas where an element can be dropped during drag and drop operations.

Reviewer 2: > Could you add a link (here in the review) to support this description? That would be quite helpful.

I descript it by myself. There are some reference documents.
[line 1306 in https://searchfox.org/mozilla-central/source/__GENERATED__/layout/style/ServoStyleConsts.h#1307](https://searchfox.org/mozilla-central/source/__GENERATED__/layout/style/ServoStyleConsts.h#1307)

![1683333364833](https://user-images.githubusercontent.com/31983543/236588749-b5ccd87a-ee6b-412c-bbf2-4b768e77863f.jpg) in [http://help.dottoro.com/lanifqvh.php](http://help.dottoro.com/lanifqvh.php)

>   - : Specify the areas where an element can be dropped during drag and drop operations.

Reviewer: :+1: Just tweak the wording per the code comment.
```suggestion
  - : Highlight color for regions where an element can be dropped during a drag-and-drop operation.
```
